src/composables/types/diagnosis.ts

```diff
@@ -0,0 +1,82 @@
+import { 
+  AcutenessType,
+  AgeRequestType,
+  ExtrasType,
+  EvidenceType,
+  SexType,
+  PrevalenceType,
+  SeverityType,
+  TriageLevelType,
+} from '@/composables/types';
+
+export type ConditionDetailsModelCategoryType = {
+  id?: string,
+  name?: string,
+}
+
+export type ConditionDetailsModelType = {
+  icd10_code?: string,
+  category?: ConditionDetailsModelCategoryType,
+  prevalence?: PrevalenceType,
+  severity?: SeverityType,
+  acuteness?: AcutenessType,
+  triage_level?: TriageLevelType,
+  hint?: string,
+  has_patient_education?: boolean,
```

Comment:
Are these properties really all **optional**? In other words: is it possible, to have the ConditionDetails object that is totally empty? Your type claims positive - it does. Is that true?

Reply:
Yes, there are all optional.
You can check Models in [API Reference](https://developer.infermedica.com/docs/v3/api) endpoint `POST /diagnosis`

<img width="372" alt="image" src="https://github.com/infermedica/boilerplate/assets/37744371/369d475e-e12a-497c-bc16-3e934ed4e045">